theano/tests/test_flake8.py

```diff
@@ -72,14 +72,6 @@
     "tensor/tests/test_blas_scipy.py",
     "tensor/tests/test_mpi.py",
     "tensor/nnet/__init__.py",
-    "tensor/nnet/tests/__init__.py",
```

Comment:
I agree with @abergeron.

On Wed, Jul 13, 2016 at 12:52 PM, abergeron <notifications@github.com>
wrote:

> In theano/tests/test_flake8.py
> <https://github.com/Theano/Theano/pull/4592#discussion_r70663827>:
>
> > @@ -72,14 +72,6 @@
> >      "tensor/tests/test_blas_scipy.py",
> >      "tensor/tests/test_mpi.py",
> >      "tensor/nnet/__init__.py",
> > -    "tensor/nnet/tests/__init__.py",
>
> Leave the __init__ file in the whitelist. It's too hard to clean up
> properly.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/Theano/pull/4592/files/9f58f5642b32173569c4e89559d26c79066f7145..f2d505c391ae7628f41d56921eeaa689af16c225#r70663827>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALC-1KxPuOSHsSz4GBsiq_HUiNjTWbtks5qVRfegaJpZM4Iu2Js>
> .
>